contracts/contracts/ResolutionRegistry.sol

```diff
@@ -21,6 +21,8 @@ contract ResolutionRegistry is CrossDomainRestrictedCalls {
     ) external restricted(resolutionChainId, msg.sender) {
         require(fillers[fillHash] == address(0), "Resolution already recorded");
         fillers[fillHash] = filler;
+        // Revert fill hash invalidation, fill proofs outweigh an invalidation
+        invalidFillHashes[fillHash] = false;
```

Comment:
Why do we need this? Resolution should work either way.

If we leave this in, this won't we idempotent any more, e.g. receiving the fill proof after the non-fill proof will look different than receiving the non-fill proof after the fill proof.